src/app/controllers/admin-forms.server.controller.js

```diff
@@ -715,5 +715,38 @@ function makeModule(connection) {
         },
       )
     },
+    transferOwner: async function (req, res) {
+      const newOwnerEmail = req.body.email
+
+      // Check if new owner's email is specified
+      if (_.isEmpty(newOwnerEmail)) {
+        const errMsg = 'Error: email should exists in req body'
+        logger.error(
+          `formId="${req.form._id}", ip=${getRequestIp(
+            req,
+          )}, message="${errMsg}"`,
+        )
+        return res.status(HttpStatus.BAD_REQUEST).send({ message: errMsg })
+      }
+
+      // Transfer owner and Save the form
+      try {
+        await req.form.transferOwner(req.session.user, newOwnerEmail)
+      } catch (err) {
+        logger.error(
+          `formId="${req.form._id}", ip=${getRequestIp(req)}, message="${
+            err.message
+          }"`,
+        )
+        return res.status(HttpStatus.CONFLICT).send({ message: err.message })
```

Comment:
i don't think it's a good practice to propagate internal database errors to the client. perhaps something that's more user friendly?

Reply:
will not save the form is `req.form.transferOwner` anymore